### PR TITLE
Publishing API spec

### PR DESCRIPTION
**Pull request by NC-albur**
**NOTE: Will re-target main with this change when PIL_1271_UKTR_submission_spec is merged in ** 

A small PR to publish the API spec for UKTR submissions to the API platform.
Spec has been generated by running `sbt routesToYamlOas` and verified by running `sbt validateOas` as well as checking the format using https://editor.swagger.io/

### CODE REVIEW

**Review by JamesMMiller — DISMISSED** [review]

File: resources/public/api/conf/1.0/application.yaml [hunk 1]
```diff
@@ -1,49 +1,135 @@
-openapi: "3.0.3"
-
+openapi: 3.0.3
 info:
-  version: 1.0.0
+  version: 0.13.0
```
Comment (JamesMMiller) [inline resources/public/api/conf/1.0/application.yaml:3]: just a question, where is this version coming from?

Comment (NC-albur) [reply resources/public/api/conf/1.0/application.yaml:3]: It looks like it's set from here:
image
Not entirely sure where this version is/should be set though, might be missing something in the build.sbt? I can have a look to find out exactly what populates this and change it from this arbitrary value

Comment (NC-albur) [reply resources/public/api/conf/1.0/application.yaml:3]: Confirmed, adding in 
`version := "1.0.0"`
To the build.sbt changes this value. Will add this value & regenerate the spec now.

Comment (JamesMMiller) [reply resources/public/api/conf/1.0/application.yaml]: Awesome, although considering we are not in prod perhaps we should have a 0.1.0 version. @ridow-hmrc, what version do you think?

Comment (JamesMMiller) [reply resources/public/api/conf/1.0/application.yaml]: It also brings up a question of how we manage minor and major changes.. do we have to provide a changelog or something?

Comment (NC-albur) [reply resources/public/api/conf/1.0/application.yaml]: Well it does make an interesting point that it's taking the version of the spec from the version of the _repository_ - so say we have more than 1 API in the repo (Which we will when we have create & amend), do _all_ API version's increment with a changing the version?

Comment (NC-albur) [reply resources/public/api/conf/1.0/application.yaml]: Discussed this point with Richard, the versioning comes from the git tags versioning and so will get incremented with releases. It just so happened to be at 0.13.0-SNAPSHOT version when I'd created the spec.
As a result, i've removed the explicit `version := "1.0.0` from the build.sbt

**Review by ridow-hmrc — APPROVED** [review]

**Pull request merged**